src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: transform ease .3s;
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The scale transformation on hover can cause the element to overflow its container. It's recommended to handle this case.